spec/requests/tickets_spec.rb

```diff
@@ -0,0 +1,201 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe '/tickets', type: :request do
```

Comment:
I love these tests! They're super cool — I think I'll follow this format from now on

Reply:
Ahaha I mostly just followed what the rails scaffolding gave me! The designers know what they're doing 😎